source/_includes/common-tasks/enable_i2c.md

```diff
@@ -1,23 +1,25 @@
 ## Enable I2C
 
-Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi.
+Home Assistant using the Home Assistant Operating System is a managed environment, which means you can't use existing methods to enable the I2C bus on a Raspberry Pi. In order to enable I2C devices you will have to 
+- Enable I2C for the Home Assistant Operating System 
+- Enable Raspberry Hats including the I2C integration
+- Setup I2C sensors
 
-### Step by step instructions
+### Step by step instructions to enable I2C for the Home Assistant Operating System
```

Comment:
This page is about the Home Assistant Operating System there is no need to put that in a big header again. Please revert this line change.